templates/base/base.html

```diff
@@ -7,37 +7,43 @@
         <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
         <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.2/font/bootstrap-icons.css">
         <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/base/base.css') }}"/>
+        <link rel="icon" type="image/x-icon" href="{{ url_for('static', filename='imgs/favicon.ico') }}">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Comfortaa:wght@600&display=swap" rel="stylesheet">
         {% block headextras %}
         {% endblock %}
 
     </head>
 
     <body>
+        {% block scriptextras %}
```

Comment:
Basing on my knowledge, scripts should be loaded at the bottom of the document.
Correct me if I'm wrong

Reply:
You're right, i just wanted to put navbar active page script together with its html file, but have to load it after bootstrap bundle does beacuse of my css based styling, gonna seperate these instead and it should do :>